playbooks/install_os.yml

```diff
@@ -17,12 +17,29 @@
         tasks_from: install.yml
       tags:
         - edpm_sshd
+    - name: Set up time services
+      when: timesync_ntp_servers is defined
+      block:
+        - name: Set FQCN for the timesync role
+          ansible.builtin.set_fact:
+            timesyncfqcn:
+              "{%- if ansible_facts['distribution'] == 'RedHat' -%}
+                redhat.rhel{%- else -%}
+                fedora.linux{%- endif -%}
+              _system_roles.timesync"
+          delegate_to: localhost
+        - name: Install and configure time service using timesync system role
+          ansible.builtin.include_role:
+            name: "{{timesyncfqcn}}"
+          tags:
+            - edpm_chrony
```

Comment:
I'd remove `edpm_chrony` tag with something like `timesync` or `chrony`. I don't think that `edpm_` prefix is mandatory for the tags.

Reply:
`dataplane_chrony` @slagle @fultonj wdyt?

Reply:
I'm fine with the tag being called `edpm_chrony` since it involes calling the edpm_chrony role. I expect the calls to that role and the tag will be removed soon so I don't think this needs further revision on that level.

Reply:
Ok, then `edpm_chrony` when it calls the `edpm_chrony`, otherwise the new tag.